two_factor/gateways/twilio/gateway.py

```diff
@@ -61,10 +66,16 @@ def send_sms(self, device, token):
             'two_factor/twilio/sms_message.html',
             {'token': token}
         )
+        messaging_service_sid = getattr(settings, 'TWILIO_MESSAGING_SERVICE_SID', None)
+        if messaging_service_sid is not None:
+            sender_kwargs = {'messaging_service_sid': messaging_service_sid}
+        else:
+            sender_kwargs = {'from_': getattr(settings, 'TWILIO_CALLER_ID')}
+
         self.client.messages.create(
             to=device.number.as_e164,
-            from_=getattr(settings, 'TWILIO_CALLER_ID'),
-            body=body)
+            body=body,
+            **sender_kwargs)
```

Comment:
This is a bit awkward to read, why not start with `send_kwargs = {"body": body}` and then add `from_` or `messaging_service_sid` as necessary. `self.client.messages.create` can be one line then too.

Reply:
Fair enough, changed it up to setup the kwargs

Reply:
:+1: